platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/RichTextEditor.kt

```diff
@@ -106,12 +120,51 @@ fun RichTextEditor(
         },
         update = { view ->
             view.setStyleConfig(style.toStyleConfig(view.context))
-            view.setTextColor(style.text.color.toArgb())
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
-                val cursorDrawable = ContextCompat.getDrawable(view.context, R.drawable.cursor)
-                cursorDrawable?.setTint(style.cursor.color.toArgb())
-                view.textCursorDrawable = cursorDrawable
+            view.applyStyle(style)
+        }
+    )
+}
+
+@Composable
+private fun PreviewEditor(
+    state: RichTextEditorState,
+    modifier: Modifier = Modifier,
+    style: RichTextEditorStyle = RichTextEditorDefaults.style(),
+) {
+    if (!LocalInspectionMode.current) {
+        throw IllegalStateException("PreviewEditor should only be used in preview mode")
+    }
+
+    val context = LocalContext.current
+
+    AndroidView(
+        modifier = modifier,
+        factory = {
+            val view = AppCompatEditText(context).apply {
```

Comment:
Why not just a material `TextField` ? It'd be a one-liner

Reply:
I wanted the preview editor to match the real editor as closely as possible. Using an `EditText` for the preview allows us to make changes to the real editor that will automatically get reflected in the preview, whereas using `TextField` would make it harder to get it looking pixel perfect.